Document surface obs xarray/nc format (#346)
NOAA-CSL/MELODIES-MONET: docs/develop/datasets.rst

Instructions for reading in aircraft and satellite observations are under development. 

If you are interested in converting a new observational dataset to our netCDF format,
please see the notes below.

* The dataset should have these dimensions (in this order):

  - ``time``
  - ``y`` (an optional singleton dimension, included for consistency with
    model surface datasets)
  - ``x`` (the site dimension)

* The dataset should have these coordinate variables:

>   - ``time`` (UTC time, as timezone-naive ``datetime64`` format in xarray: ``time`` dim)

Author: ```suggestion
  - ``time`` (UTC time, as timezone-naive ``datetime64`` format in xarray; ``time`` dim)
```

  - ``siteid`` (unique site identifier, as string; ``x`` dim)
  - ``latitude`` (site latitude, in degrees; ``x`` dim)
  - ``longitude`` (site longitude, in degrees; ``x`` dim)

* This variable is required for regulatory metrics:

>   - ``time_local`` (local time, as timezone-naive ``datetime64`` format in xarray;
>     note that this varies in both the ``time`` and ``x`` dimensions)

Author: ```suggestion
  - ``time_local`` (local time,
    usually local standard time, not including daylight savings,
    as timezone-naive ``datetime64`` format in xarray;
    note that this varies in both the ``time`` and ``x`` dimensions)
```

* It's good practice to include ``units`` attributes for your data variables,
  though this is not strictly required.
  Similarly, you may wish to include ``long_name``\ s.

* Site metadata variables (e.g. site name, site elevation, EPA region, etc.)
  should ideally be stored as varying only in the ``x`` dimension, to save space.

* If you have sub-hourly data, you may want to aggregate it to hourly,
  especially if different sites have different time resolutions.

Example abbreviated xarray representation for AirNow
demonstrating these qualities:

.. code-block:: text

   <xarray.Dataset>

>    Dimensions:     (time: 867, y: 1, x: 2231)
>    Coordinates:
>      * time        (time) datetime64[ns] 2023-04-04 ... 2023-04-16T00:30:00

Author: to hourly
```suggestion
   Dimensions:     (time: 289, y: 1, x: 2231)
   Coordinates:
     * time        (time) datetime64[ns] 2023-04-04 ... 2023-04-16
```

Instructions for reading in aircraft and satellite observations are under development. 

If you are interested in converting a new observational dataset to our netCDF format,
please see the notes below.

* The dataset should have these dimensions (in this order):

  - ``time``
  - ``y`` (an optional singleton dimension, included for consistency with
    model surface datasets)
  - ``x`` (the site dimension)

* The dataset should have these coordinate variables:

  - ``time`` (UTC time, as timezone-naive ``datetime64`` format in xarray; ``time`` dim)
  - ``siteid`` (unique site identifier, as string; ``x`` dim)
  - ``latitude`` (site latitude, in degrees; ``x`` dim)
  - ``longitude`` (site longitude, in degrees; ``x`` dim)

> * This variable is required for regulatory metrics, ,
>   and can be optionally used for general timeseries.
>   If you don't care about those, you might omit it:

Author: ```suggestion
* This variable is required for regulatory metrics,
  and can be optionally used for time series plots.
  Otherwise, you might omit it:
```
@blychs what do you think?

Reviewer: I agree it's better.

  - ``latitude`` (site latitude, in degrees; ``x`` dim)
  - ``longitude`` (site longitude, in degrees; ``x`` dim)

> * This variable is required for regulatory metrics:

Reviewer: I just tried to include @zmoon's comment, it shows up in the file but I don't know why it does not show up here.

> * This variable is required for regulatory metrics:

Author: Here it's linked to the state at the time you made the comment. You can see this comment is now marked "Outdated".

  - ``latitude`` (site latitude, in degrees; ``x`` dim)
  - ``longitude`` (site longitude, in degrees; ``x`` dim)

* This variable is required for regulatory metrics, ,
  and can be optionally used for general timeseries.

>   If you don't care about those, you might omit it:

Reviewer: Include mention to timeseries
